perl-package/AI-MXNetCAPI/mxnet.i

```diff
@@ -0,0 +1,1558 @@
+%module "AI::MXNetCAPI"
```

Comment:
There's a lot of custom names embedded into swig interface files (that have a meaning for swig and route it to a specific chunk of a perl code to  inject) and there's no clean way to do it programmatically, for example there's no clean way of knowing how to handle new variable added to a function (should it be input or output or neither).
Should API change, its better for the tests to fail and me reworking the affected part in the correct way. At least until char-level RNNs get to the point of average Perl programmer, my testing indicate that they are not yet ready :-).